content/en/monitors/service_level_objectives/error_budget.md

```diff
@@ -38,6 +38,7 @@ error_budget("slo_id").over("time_window") > 75
 
 In addition, SLO error budget alerts can also be created using the [datadog_monitor resource in Terraform][6]. Below is an example `.tf` for configuring an error budget alert for a metric-based SLO using the same example query as above.
 
+For provider version v2.7+ :
```

Comment:
Should this be "For provider versions 2.7 and earlier" instead? (currently implies and later). Or maybe v3.0 and earlier?

Reply:
Oh thanks, it should be : For provider versions v2.7.0 or earlier and v2.13.0 or later. I'll fix this

Reply:
I've fixed this error, and added bold modifier to help clarify the doc

Reply:
🤔 thinking about it, maybe it should be versions prior to v3.0, not sure to know which one would be best understood

Reply:
Thanks! I think this is better now.